applications/GeoMechanicsApplication/custom_workflows/dgeosettlement.cpp

```diff
@@ -11,17 +11,18 @@
 //
 #include "dgeosettlement.h"
 #include "input_output/logger.h"
-#include "custom_utilities/input_utilities.h"
 #include "utilities/variable_utils.h"
-#include "includes/model_part_io.h"
-
+#include "custom_utilities/input_utility.h"
+#include <iostream>
```

Comment:
I don't think we need this include, do we?

Reply:
Good catch, removed